src/react-app-env.d.ts

```diff
@@ -45,3 +45,11 @@ declare module '*.svg' {
   const src: string;
   export default src;
 }
+
```

Comment:
@stephanoshadjipetrou is this absolutely necessary? It causes an error in every file that has an import from the "react" module. See image for example
<img width="725" alt="image" src="https://github.com/user-attachments/assets/10993c4a-fa94-4c7b-8a75-fbcbc8242e57">

Reply:
@okorie2 if you remove this then it works ok?